src/index.js

```diff
@@ -117,14 +117,20 @@ export default class Bot extends EventEmitter {
    *                           gets the update
    * @return {object} returns the bot object
    */
-  command(command, listener) {
+  command(command, customMessage, listener) {
```

Comment:
I'm not in favor of optional arguments that come in the middle and not the end.
Let's move this to the last argument to avoid confusions.